profitbricks/README.md

```diff
@@ -0,0 +1,64 @@
+# jclouds ProfitBricks
+
+## Terms
+Like any cloud provider, ProfitBricks has its own set of terms in cloud computing. To abstract this into jclouds' Compute interface, these terms were associated:
+
+- Node - composite instance of `Server` and `Storage`
+- Image - both *user-uploaded* and *provided* `Images`; and `Snapshots`
+- Location - `DataCenters` and `Region` (Las Vegas, Frankfurt, etc.)
+- Hardware - number of cores, RAM size and storage size
+
+## Getting Started
+
+Assuming that there's **atleast one** datacenter existing in your account, the provider needs only an *identity* (your ProfitBricks email), and *credentials* (password) to provision a `Node`, by using a ProfitBricks-provided ubuntu-12.04 image as a template. 
+
+```java
+ComputeService compute = ContextBuilder.newBuilder( "profitbricks" )
+					.credentials( "profitbricks email", "password" )
+					.buildView( ComputeServiceContext.class )
+					.getComputeService();
+```
+
+
+This works well; however, we won't be able to use jclouds' ability to execute *scripts* on a remote node. This is because, ProfitBricks' default images require users to change passwords upon first log in.
+
+To enable jclouds to execute script, we need to use a custom image. The easiest way to do this is via ProfitBricks snapshot:
+
+-  Go to your [DCD](https://my.profitbricks.com/dashboard/).
+-  Provision a server + storage, and connect it to the internet. Upon success, you will receive an email containing the credentials needed to login to your server.
+-  Login to your server, and change the password, as requested.
+
+```
+~ ssh root@<remote-ip>
+...
+Changing password for root.
+(current) UNIX password: 
+Enter new UNIX password: 
+Retype new UNIX password: 
+~ root@ubuntu:~# exit
+
+```
+
+- Go back to the DCD, and *make a snapshot* of the storage. Put a descriptive name.
+- Configure jclouds to use this *snapshot*.
+
+```java 
+Template template = compute.templateBuilder()
+	.imageNameMatches( "<ideally-unique-snapshot-name>" )
+	.options( new TemplateOptions()
```

Comment:
No, just to substitute the "new". Don't encourage users to build a new template options object; it should be better retrieved from the ComputeService, as it will provide the provider-specific implementation allowing users to use the `.as(ProfitBricksTempalteOptions.class)` when needed.